js/src/provider/WalletLinkSdkUI.ts

```diff
@@ -58,6 +58,24 @@ export class WalletLinkSdkUI extends WalletLinkUI {
   }
 
   // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
```

Comment:
Why is `ts-ignore` needed here?

Reply:
if i understand it correctly, since addEthereumChain in WalletLinkSdkUI is not doing anything, if without this, the ts lint would warn that the parameter declared but not used.